src/services/transaction-cron.ts

```diff
@@ -643,7 +644,7 @@ export default class TransactionCron {
           amount: new BigNumber(
             value.message.fungibleToken.amount,
             16
-          ).toString(),
+          ).toFixed(),
```

Comment:
A reasonable question:
Why are we using BigNumber API instead of BigInt for the `prettifiedAmount` field in some particular code snips?
cc: @0xSasaPrsic @rac-sri

Reply:
@rac-sri  any thoughts, can we just change that in `prettifiedAmount` without changing the amount in the DB, not sure if the UI will break with nonscientific notation for the amount field.

Reply:
> not sure if the UI will break with nonscientific notation for the amount field

For instance [this turing tx](https://app.datadoghq.com/logs?query=kube_app_instance%3Aturing-bridge-indexer%20MessageSent&agg_m=count&agg_m_source=base&agg_t=count&cols=host%2Cservice&event=AgAAAZEnpeGIhVsmZAAAAAAAAAAYAAAAAEFaRW5wZVhWQUFCMV9Zbzg4ci01cFFBSgAAACQAAAAAMDE5MTI3YTktZGE1OC00YTIzLTkwYmYtOTM5MWZkN2MwNjdh&fromUser=true&messageDisplay=inline&refresh_mode=paused&storage=hot&stream_sort=desc&viz=stream&from_ts=1722945166274&to_ts=1722983268000&live=false) was persisted with nonscientific notation. We could just check on turing bridge for the given address.

What I don't know is the opposite: what are the impacts on scientific notation values persisted on amount field? Will Frontend will be able to parse them? If we're using this very same BigNumber lib we could migrate to toFixed on frontend side as well (if not migrated yet).

IMHO biggest problem relies on non deterministic behavior in `BigNumber().toString()`, that sometimes will return a nonscientific notation value, sometimes will return a scientific notation value; depending on the total digits a number has.

Reply:
of-course we can. Prettified amount is generated during logging only. Or am i misunderstanding the question here. Prettified amount is nowhere stored in the db.

Reply:
The prettified amount is just for logging and not stored anywhere.